src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("---------------------------------------\n");
+        System.out.println("What's up! I'm Duke\nPlease feed be commands :)" );
+        System.out.println("---------------------------------------" );
+
+        Scanner scan = new Scanner(System.in);
+        String input = " ";
+
+//        Initialize Task container
+        ArrayList<Task> tasks = new ArrayList<>();
+        int taskNumber = 0;
+
+
+        while(!input.equals("bye")) {
+            input = scan.nextLine().trim();
+            String command = input.split(" ")[0];
+
+            switch (command) {
+                case "bye":
+                    System.out.println("\n---------------------------------------" );
+                    System.out.println("Bye. Sayonara and goodbye!");
+                    System.out.println("---------------------------------------" );
+                    break;
+
+                case "list":
+//                    Numbers should change accordingly when deleted (For future Ref.)
+                    System.out.println("\n---------------------------------------" );
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println(String.valueOf(i+1) + "." + tasks.get(i).toString());
+                    }
+                    System.out.println("---------------------------------------" );
+                    break;
+
+                case "done":
+//                    Possible Error: index provided is out of bounds (NullPointerException)
+                    int index = Integer.parseInt(input.split(" ")[1]) - 1;
+                    tasks.get(index).markDone();
+
+                    System.out.println("\n---------------------------------------" );
+                    System.out.println("Nice! I've marked this task as done:");
+                    System.out.println("[" + tasks.get(index).getStatusIcon() + "] " + tasks.get(index).name.trim());
+                    System.out.println("---------------------------------------" );
+                    break;
+
+                case "todo":
+                    try {
+                        String name = getTodoName(input);
+                        Todo todo = new Todo(name);
+                        tasks.add(todo);
+
+                        System.out.println("\n---------------------------------------" );
+                        System.out.println("Got it. I've added this task:");
+                        System.out.println(todo.toString());
+                        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+                        System.out.println("---------------------------------------" );
+                    } catch(DukeException e) {
```

Comment:
I guess a white space can be placed here after `catch`, as per coding standards. 🤔